notebooks/official/migration/sdk-custom-xgboost-prebuilt-container.ipynb

```diff
@@ -1062,16 +959,9 @@
         "id": "make_online_predictions:migration"
       },
       "source": [
-        "## Make online predictions"
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "deploy_model:migration,new,mbsdk"
-      },
-      "source": [
-        "### [predictions.deploy-model-api](https://cloud.google.com/vertex-ai/docs/predictions/deploy-model-api)"
+        "### Generate online predictions\n",
+        "\n",
+        "To learn more of online predictions refer, [Overview of getting predictions on Vertex AI](https://cloud.google.com/vertex-ai/docs/predictions/deploy-model-api)."
```

Comment:
URL involves a redirect.
- change: https://cloud.google.com/vertex-ai/docs/predictions/deploy-model-api
- to: https://cloud.google.com/vertex-ai/docs/predictions/overview